src/scss/atomic/organisms/_hero-video.scss

```diff
@@ -0,0 +1,47 @@
+.hero__video {
+    @extend .organism;
+    @extend %reset-last-margin;
+    position: relative;
+
+    &--reset p:last-child {
+        margin-bottom: 0;
+    }
+
+    .hero__headline,
```

Comment:
you don't have to add your css classes behind .hero__video it can be at the root of the css file because we're using BEM: 

.hero__video {
}
.hero__image {
}